### DISCUSSION TITLE
DeepEdit: There is no change in the results after changing the foreground simulation points in the inference code

### DISCUSSION THREAD

**Discussion by praveenrp295** [Project-MONAI/MONAI#6579] [Q&A]
Problem: I am trying to improve the segmentation of an image by providing simulation points in the inference code ("LV": [[80, 57, 35]]). I have trained the model using the train.py script provided in the repository, and I am using the same model to predict the mask. However, some parts of the image are not segmented correctly, so I provided the location points as a list in the inference code ("LV": [[80, 57, 35]]). Unfortunately, the location of the mask is not improving in the output.

Need help. I am using deepedit algo.

**Comment by Nic-Ma** [comment]
Hi @praveenrp295 ,

Could you please help share more clear info about which tutorial or example you refer to?
And @tangy5 can help with further analysis.

Thanks.

**Reply by praveenrp295** [reply]
I am referring to deepedit tutorial @tangy5 and @Nic-Ma

**Reply by praveenrp295** [reply]
Hi @Nic-Ma Could you please help in this regard.

**Reply by Nic-Ma** [reply]
Hi @diazandr3s , could you please help take a look at this question?

Thanks in advance.

**Reply by diazandr3s** [reply]
Thanks for the ping @Nic-Ma.
@praveenrp295, is this for left ventricle segmentation? how many samples are you using to train the DeepEdit model?
Are you using the default image size to train the model (128x128x128)? What is the minimum image size in your dataset?

**Reply by praveenrp295** [reply]
Yes, Model is for LV segmentation and using around 50 subjects.  Using Train script provided by deep edit model. Yesa using default image size. minimum image size is <121*121*100>

**Reply by praveenrp295** [reply]
It would be good if you provide detailed documentation on the deepedit, Have few questions on deepedit.
1. How lost of points make impact on the mask?
2. How many minimum points required? As it changes the location of point or it grows along adjacent slices?
3. Is it works on well trained model or not?
4. How list of background points to inference works? Is it erases pixels? OR we should give both foreground and background to get efficient annotation?
